src/achievement.cpp

```diff
@@ -124,11 +124,11 @@ std::string enum_to_string<requirement_visibility>( requirement_visibility data
 
 } // namespace io
 
-static nc_color color_from_completion( bool is_conduct, achievement_completion comp )
+static nc_color color_from_completion( bool is_conduct, achievement_completion comp, bool is_title )
 {
     switch( comp ) {
         case achievement_completion::pending:
-            return is_conduct ? c_light_green : c_yellow;
+            return is_conduct ? c_light_green : ( is_title ? c_yellow_magenta : c_light_cyan );
```

Comment:
I'm not a big fan of the yellow_magenta look, but I don't have a specific alternative to propose.

Reply:
Maybe just use white text for the titles?  That seems to be the convention in most of the game.